docs/antrea-network-policy.md

```diff
@@ -86,7 +86,7 @@ Tier CRD object, that will be enforced after developer-created K8s NetworkPolici
 The details for these tiers are shown below:
 
 ```text
-    Emergency   -> Tier name "emergency" with priority "5"
+    Emergency   -> Tier name "emergency" with priority "20"
```

Comment:
Update the tier priority changes for the following tiers as well?

Reply:
done